.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,21 +1,31 @@
 ---
-name: 🐛 Bug Report
-about: Report a reproducible bug or regression in the core React Native library.
+name: I want to report an issue with React Native.
+about: You want to report a reproducible bug or regression in React Native.
 ---
 
-<!-- Requirements: please go through this checklist before opening a new issue -->
-  - [ ] Review the documentation: https://facebook.github.io/react-native
-  - [ ] Search for existing issues: https://github.com/facebook/react-native/issues
-  - [ ] Use the latest React Native release: https://github.com/facebook/react-native/releases
+<!--
+  If you are looking for help, please review the documentation or ask a question on Stack Overflow:
+  - https://facebook.github.io/react-native
+  - https://stackoverflow.com/questions/tagged/react-native
+
+  If you want to report a problem with React Native itself, please fill out the template.
+
+  If you would like to report an issue in the documentation, or anything related to the React Native
+  website, please visit https://github.com/facebook/react-native-website/issues.
+-->
 
 ## Environment
-Run `react-native info` in your terminal and paste its contents here.
+<!-- Run `react-native info` in your terminal and paste its contents here. -->
 
 ## Description
-Describe your issue in detail. Include screenshots if needed. If this is a regression, let us know.
+<!--
+  Describe your issue in detail. Include screenshots if needed. If this is a regression, let us know.
+  -->
 
 ## Reproducible Demo
-Let us know how to reproduce the issue. Include a code sample, share a project, 
-or share an app that reproduces the issue using https://snack.expo.io/. 
-If you run into an Android or JS build issue, please consider adding CI to your project. Check out https://github.com/react-native-community/ci-sample for how to do this.
```

Comment:
@gengjiawen I noticed you added this item recently. I am not sure the template is the best place to include this. I'll remove it for now, but happy to discuss further about what you'd like to encourage people to do.